lib/cassandro/model.rb

```diff
@@ -230,6 +230,14 @@ def self.query(where, *values)
       Cassandro.client.execute(st, *values)
     end
 
+    def self.count
```

Comment:
You're overwriting an existing method https://github.com/tarolandia/cassandro/blob/add_count_method/lib/cassandro/model.rb#L207